api/v1beta1/machine_types.go

```diff
@@ -97,6 +97,11 @@ type MachineSpec struct {
 	// +optional
 	NodeDrainTimeout *metav1.Duration `json:"nodeDrainTimeout,omitempty"`
 
+	// VolumeDetachTimeout is the total amount of time that the controller will spend on waiting for a volume
+	// to be detached. The default value is 0, meaning that the volume can be detached without any time limitations.
```

Comment:
How would I say that I don't want to wait at all? Is that not what 0 should mean?

Reply:
You can set nodeDrainTimeout > 0, so we do not wait at all for the volumes to be detached

Reply:
> if NodeDrain is greater than 0 and the other isn't, NodeDrain takes precedence and will overrule the detach timeout

Exactly, that is the point, sorry that I did not make it clear, will update the description.